### PR TITLE
config/update_plugins

### PR DESCRIPTION
**Pull request by sebascm**
- Update the dependencies and clean unnecessary ones (if it is needed).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>1.5.1.RELEASE</version>
+    <version>1.5.22.RELEASE</version>
```
Comment (sebascm) [inline pom.xml:14]: It can't be updated to 2.X version because it fails during the compilation

**Review by Chusca — DISMISSED** [review]

File: pom.xml (hunk 1 above)
Comment (franrua) [reply pom.xml:14]: It doesn't matters, I've seen it on wiki, it's ok.

**Review by franrua — CHANGES_REQUESTED** [review]
The plugins are updated and the build is done without problems. But you must specify maven minimal version. You can check it with command `mvn versions:display-plugin-updates`

File: pom.xml [hunk 2]
```diff
@@ -50,6 +50,9 @@
   </dependencies>
   <packaging>pom</packaging>
 
+  <prerequisites>
```
Comment (sebascm) [inline pom.xml]: Even with maven-enforcer-plugin, Maven is complaining that prerequisites is missing

Comment (franrua) [reply pom.xml]: Yes, is a [known problem](https://github.com/mojohaus/versions-maven-plugin/issues/48). Pablo also suffered this issue. Ask him help

**Review by franrua — APPROVED** [review]

**Comment by franrua** [comment]
The plugins are updated, we will take care of this issue later, merge the branch.

**Review by Chusca — APPROVED** [review]

**Pull request merged**